### PR TITLE
COPS-7168 - backport 2 pages to Kommander 2.1

### PR DESCRIPTION
**Pull request by Colin1841**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7168

## Description of changes being made
Backporting these 2 pages from 2.2 into the 2.1 doc set; no other changes.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kommander/2.1/install/mgmt-cluster-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+This section only details requirements for management cluster-specific applications. For the list of all platform applications, see [Platform Application Configuration Requirements](../../workspaces/applications/platform-applications/platform-application-requirements/).
+
+This table describes the workspace platform applications specific to the management cluster, minimum resource requirements, and minimum persistent storage requirements.
+
+| Common Name       | Application Name          | Minimum Resources Suggested | Minimum Persistent Storage Required |
+|-------------------|---------------------------|-----------------------------|-------------------------------------|
+| Grafana           | centralized-grafana       | cpu: 200m<br>memory: 100Mi  |                                     |
+| Kubecost          | centralized-kubecost      | cpu: 1200m<br>memory: 4151Mi | # of PVs: 1<br>PV sizes: 32Gi      |
+| Chartmuseum       | chartmuseum               |                             | # of PVs: 1<br>PV sizes: 2Gi        |
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/install/mgmt-cluster-apps/index.md:18]:
```suggestion
```
this was introduced in 2.2

File: pages/dkp/kommander/2.1/install/mgmt-cluster-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: layout.pug
+navigationTitle: Management cluster application requirements
+title: Management cluster application requirements
+menuWeight: 40
+excerpt: Management cluster application minimum resources and storage requirements
+beta: false
+enterprise: true
+---
+This section only details requirements for management cluster-specific applications. For the list of all platform applications, see [Platform Application Configuration Requirements](../../workspaces/applications/platform-applications/platform-application-requirements/).
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/install/mgmt-cluster-apps/index.md:10]: Just a total nit

```suggestion
---

This section only details requirements for management cluster-specific applications. For the list of all platform applications, see [Platform Application Configuration Requirements](../../workspaces/applications/platform-applications/platform-application-requirements/).
```

File: pages/dkp/kommander/2.1/workspaces/applications/platform-applications/platform-application-requirements/index.md [hunk 3]
```diff
@@ -29,7 +31,7 @@ The following table describes all the workspace platform applications that are a
 | kube-oidc-proxy |  |  | Yes |
 | kubecost | cpu: 700m<br />memory: 1700Mi | # of PVs: 3<br />PV sizes: 2Gi, 32Gi, 32Gi (total: 66Gi) | Yes |
 | kubernetes-dashboard | cpu: 250m<br />memory: 300Mi |  | Yes |
-| logging-operator | cpu: cpu: 350m * # of nodes + 600m<br />memory: 228Mi + 350Mi * # of nodes | # of PVs: 1<br />PV sizes: 10Gi | No |
+| logging-operator | cpu: 350m * # of nodes + 600m<br />memory: 228Mi + 350Mi * # of nodes | # of PVs: 1<br />PV sizes: 10Gi | No |
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/platform-applications/platform-application-requirements/index.md:34]: NICE catch!

**Review by msdolbey — APPROVED** [review]
👍 
Yarp

**Review by cmastr — APPROVED** [review]

**Pull request merged**